manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PathofBuildingCommunity.PathofBuilding
+PackageVersion: 2.19.2.0

```

```suggestion
PackageVersion: 2.19.2
```
This must match Control Panel.

You'll also need to change `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0` to `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2`